### PR TITLE
Remove include source option that produced additional .nupkg

### PR DESCRIPTION
**Pull request by ahsanaman92**

### CODE REVIEW

**Comment by Numpsy** [comment]
what extra nupkg is that? (is that the snupkg symbols one, or something else?)

File: Directory.Build.props [hunk 1]
```diff
@@ -9,7 +9,6 @@
 		<RepositoryType>git</RepositoryType>
 		<RepositoryUrl>https://github.com/ahsanaman92/JpegXmpWritePluginMDE.git</RepositoryUrl>
```
Comment (Numpsy) [inline Directory.Build.props:9]: actually we shouldn't need to hard code this URL - SourceLink should be able to get the info from the git repo itself

**Comment by ahsanaman92** [comment]
> what extra nupkg is that? (is that the snupkg symbols one, or something else?)

it is JpegXmpWritePluginMDE.2.9.0.symbols.nupkg, It contains all *.cs files in src folder.

File: Directory.Build.props (hunk 1 above)
Comment (ahsanaman92) [reply Directory.Build.props:9]: Done

**Review by Numpsy — APPROVED** [review]

**Pull request merged**